src/components/port-picker/PortsInput.vue

```diff
@@ -88,6 +110,14 @@ export default {
       setShowVirtual() {
         this.showVirtual = getConfig('showVirtualMode').showVirtualMode;
       },
+      onChange(event) {
+        if (event.target.value === 'requestpermission') {
+          event.target.value = 'manual';
```

Comment:
This looks a bit fishy to me 🤔 But I might be missing something about vue myself, coming from react world this wouldn't really work.

Reply:
I did several tests with different options, and this is what worked and had better behaviour. A best solution is to add a button to request permission, but maybe is not as intuitive for the user that will go to the combo directly. As first version I think that does the job.

Reply:
Changed. I'm pretty sure it worked before, maybe some other part of my code changed it and I didn't notice it.